js/authentication.md

```diff
@@ -225,6 +225,78 @@ Auth.verifyCurrentUserAttributeSubmit(attr, 'the_verification_code')
 });
 ```
 
+#### Set Session Manually
+
+You can manually set the user session by:
+```js
+import Auth, {
+    AWSCognitoProvider, 
+    GoogleProvider,
+    FacebookProvider,
+    AmazonProvider,
+    DeveloperProvider,
+    GenericProvider,
+    SessionType
+} from '@aws-amplify/auth';
+
+const { 
+    session, // the session is either a CognitoUserSession or a FederatedProviderSession
+    user, // The user is either CognitoUser or FederatedUser
+    credentials 
+} = await Auth.setSession({
+    username: 'Alice',       // Required, user name
+    attributes: {            // Optional, user attributes
+        email: 'alice@gmail.com',
+        phone_number: '123-456-7890'
+    },
+    tokens: {                 // Either id token or access token should be provided
+        idToken: 'xxxxxxx',  // Optional, the id token
+        accessToken: 'xxxxxxx', // Optional, the access token
+        refreshToken: 'xxxxxxx', // Optional, the refresh token
+        expires_at: 1540857110755, // Required, the timestamp when the token expires
+    },
+    provider: AWSCognitoProvider.NAME, // Required, the name of the token provider
```

Comment:
Need to make this provider generic for more than just Cognito

Reply:
AWSCognitoProvider Does not exist?! 
my aws-amplify: 1.1.17